product.php

```diff
@@ -0,0 +1,185 @@
+<?php include 'includes/header.php'; ?>
+
+
+<!--import css skeleton-->
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.min.css"/>
+
+<main>
+<!--Internal Product Navigation Links-->
+    <div class="row"> <!--one-->
+        <div class="twelve columns">
+            <nav>
+                <a href="">Home</a> >>
+                <a href="">Dogs</a> >>
+                <a href="">Food</a> >>
+                <a href="">Desserts</a>
```

Comment:
Because the breadcrumbs are navigation, They should be part of a list. Please use the `<ul>` and `<li>` elements to create the list. Unordered and ordered lists are important for screen readers to keep related links together. Don't worry if it's not styled how you're envisioning. The focus this week is HTML, not CSS.